features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/tracking/NavigationViewTrackingStrategy.kt

```diff
@@ -180,14 +180,26 @@ class NavigationViewTrackingStrategy(
     @Suppress("SwallowedException")
     private fun Activity.findNavControllerOrNull(@IdRes viewId: Int): NavController? {
         return try {
-            Navigation.findNavController(this, viewId)
+            val navController = if (this is FragmentActivity) {
```

Comment:
can you name it more specifically -> fragmentNavController

Reply:
nav controller is in the fragment in any case, the difference is just that with old approach it is in user-created fragment, and with new approach it is in the lib-created fragment which is controlled by `FragmentContainerView`, so not sure if rename is worth it.